config/src/test/scala/com/typesafe/config/impl/TestUtils.scala

```diff
@@ -656,6 +657,10 @@ abstract trait TestUtils {
         tokenize(s).asScala.toList
     }
 
+    def tokenizeAsString(s: String) = {
```

Comment:
this seems like it might belong in TestUtils.scala rather than in the production code